c2corg_ui/static/js/card.js

```diff
@@ -107,4 +107,101 @@ app.CardController.prototype.createURL = function() {
 };
 
 
+/**
+ * Gets the global ratings for each activity of a route.
+ * @export
+ * @return {Object | null} ratings
+ */
+app.CardController.prototype.getGlobalRatings = function() {
+  var doc = this.doc;
+  var ratings = {};
+
+  doc.activities.forEach(function(a) {
+    if (a === 'rock_climbing' || a === 'mountain_climbing' || a === 'snow_ice_mixed') {
+      ratings['global_rating'] = doc.global_rating;
+
+    } else if (a === 'via_ferrata') {
+      ratings['via_ferrata_rating'] = doc.via_ferrata_rating;
+
+    } else if (a === 'snowshoeing') {
+      ratings['snowshoe_rating'] = doc.snowshoe_rating;
+
+    } else if (a === 'hiking') {
+      ratings['hiking_rating'] = doc.hiking_rating;
+
+    } else if (a === 'ice_climbing') {
+      ratings['ice_climbing'] = this.slashSeparatedRating(doc.ice_rating, doc.engagement_rating);
+
+    } else if (a === 'skitouring') {
+      ratings['labande_global_rating'] = this.slashSeparatedRating(doc.ski_rating, doc.ski_exposition);
+      ratings['labande_global_rating'] += doc.labande_global_rating ? doc.labande_global_rating : '';
```

Comment:
No: the blank space is added by `slashSeparatedRating` at the end of the rating.